modules/virt-enabling-volume-snapshot-boot-source.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assembly:
+//
+// * virt/storage/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-enabling-volume-snapshot-boot-source_{context}"]
+= Enabling volume snapshot boot sources
+
+Enable volume snapshot boot sources by setting the parameter in the `DataImportCron` namespace. Although `DataImportCron` was originally designed to maintain only PVC sources, volume snapshot sources scale better than PVC sources for certain storage types.
+
+[NOTE]
+====
+It is recommended to use volume snapshots on a storage profile that is proven to scale better when cloning from a single snapshot.
+====
+
+.Prerequisites
+
+* You must have access to a volume snapshot with the operating system image.
+* The storage profile must support snapshotting.
+
+.Procedure
+
+. Open the storage profile object that corresponds to the storage class used to provision boot sources.
++
+[source,terminal,subs="attributes+"]
+----
+$ oc edit storageprofile <storage_class>
+----
+
+. Review the `dataImportCronSourceFormat` specification of the `DataImportCron` namespace to confirm whether or not the VM is using PVC or volume snapshot by default.
+
+. Edit the storage profile, if needed, by updating the `dataImportCronSourceFormat` specification to `snapshot`.
++
+.Example storage profile
+[source,yaml]
+----
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: StorageProfile
+metadata:
+# ...
+spec:
+  dataImportCronSourceFormat: snapshot
+----
+
+.Verification
+
+. Open the storage profile object that corresponds to the storage class used to provision boot sources.
+
++
+[source,terminal,subs="attributes+"]
+----
+$ oc edit storageprofile <storage_class>
+----
+
+. Confirm that the `dataImportCronSourceFormat` specification of the `DataImportCron` namespace is using volume snapshots by default.
```

Comment:
I think verification that this "went through" would be to:
- Ensure the storage profile status section reflects the change (basically that `storageprofile.status.dataImportCronSourceFormat` is equal to "snapshot" now)
- Inspect the DataSource objects which the DataImportCron points to, to ensure they now reference VolumeSnapshots instead of PVCs. This may take a little longer

Also, dataImportCronSourceFormat is a specification under the StorageProfile object, not DataImportCron
so the namespace word throws me off. Am I missing something? StorageProfile is cluster wide